support/src/main/java/com/yubico/yubikit/support/DeviceUtil.java

```diff
@@ -62,7 +62,7 @@
     static Pair<Version, Optional<Integer>> readOtpData(SmartCardConnection connection)
             throws ApplicationNotAvailableException, IOException {
 
-        YubiOtpSession otpSession = new YubiOtpSession((SmartCardConnection) connection);
+        YubiOtpSession otpSession = new YubiOtpSession(connection);
```

Comment:
changes to this file are not specific to this PR, they are already in the support-module branch. please ignore